Add a test for <llama-select-fab>'s "fileselected" event

beverloo opened the pull request:
This stuff is fascinating.

First, TypeScript isn't supported yet. It's not just the types - adding those is easy, it's that Babel has to be set up to translate the test files prior to executing them. That should be done in a separate PR.

Second, while it makes sense for Puppeteer to work this way, having two JavaScript contexts in the same file is _confusing_. The event listener I add in `evaluate()` is executed in the page's context. Transferring data to the test context requires exposing a function (`setSelectedFilenameForTesting`). This seems to only support limited cloning, so I can't transfer the full event, nor the `File` instance, thus have to settle on just using the filename.

The rest is straightforward enough. I decided to upload the current file to avoid needing to check in a resource for testing purposes—the filtering logic is not ours anyway. We should figure out how to share code for some of the environment set-up across tests.

@PEConn I notice that you created a `tests/` directory. In this PR I'm adding a test to live next to the file it's testing something for, which I think is cleaner. Thoughts?

glennhartmann commented on src/components/llama-select-fab.test.js:
```diff
@@ -0,0 +1,53 @@
+
+    const selectHandle = await page.$('llama-select-fab');
+    const inputHandle = await selectHandle?.$('input[type="file"]');
+    expect(inputHandle).not.toBeNull();
```
frameworks that try to pretend they're just normal English sentences are a pet peeve of mine

glennhartmann reviewed (changes requested):
```
  32:64  error  Expected parentheses around arrow function argument  arrow-parens
  36:34  error  Expected parentheses around arrow function argument  arrow-parens
  37:54  error  Expected parentheses around arrow function argument  arrow-parens
  40:7   error  Missing semicolon                                    semi
  47:33  error  Expected parentheses around arrow function argument  arrow-parens
  48:1   error  Expected indentation of 6 spaces but found 8         indent
```

PEConn reviewed (approved):
I'm glad you managed to get Puppeteer working! Thanks for figuring out all of the exposeFunction stuff.

I agree that the tests should live alongside the code - maybe now that we have a real test that uses puppeteer, we can get rid of the one I added.

beverloo commented:
> ```
>   32:64  error  Expected parentheses around arrow function argument  arrow-parens
>   36:34  error  Expected parentheses around arrow function argument  arrow-parens
>   37:54  error  Expected parentheses around arrow function argument  arrow-parens
>   40:7   error  Missing semicolon                                    semi
>   47:33  error  Expected parentheses around arrow function argument  arrow-parens
>   48:1   error  Expected indentation of 6 spaces but found 8         indent
> ```

All amended.

> I agree that the tests should live alongside the code - maybe now that we have a real test that uses puppeteer, we can get rid of the one I added.

Good - deleted the `puppeteer.test.js` file, and moved the TODO to make sure we continue to have reference of the Puppeteer 13 issue.
